lesson01/assignment/scripts.js

```diff
@@ -1,8 +1,12 @@
 // Code that requires the DOM be loaded should not be run until the DOM is loaded
 document. addEventListener("DOMContentLoaded", function(){
 
-  // Find the example form, and handle its submit event
-  const exampleForm = document.querySelector('#example-form')
+  // Find the forms, and handle its submit event
+  const exampleForm = document.querySelector('#example-form');
+  const form1 = document.querySelector('#form1');
+  const form2 = document.querySelector('#form2');
+  const form3 = document.querySelector('#form3');
+  const form4 = document.querySelector('#form4');
```

Comment:
This code would be easier to understand if the form ids and these variables were more semantic. For example `#every-form` and `everyForm`.